WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -149,6 +150,13 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         setupTransparentNavBar()
     }
 
+    private func subscribeToContentSizeCategory() {
+        NotificationCenter.default.addObserver(self,
```

Comment:
As a good practice, what do you think about unsubscribing to notifications on `deinit`?